jobs/build.Jenkinsfile

```diff
@@ -274,10 +267,22 @@ lock(resource: "build-${params.STREAM}") {
             if (parent_version != "") {
                 parent_arg = "--parent-build ${parent_version}"
             }
-            def version = new_version ? "--version ${new_version}" : ""
+            def version_arg = ""
+            if (params.VERSION) {
+                version_arg = "--version ${params.VERSION}"
+            } else {
+                def use_versionary = pipecfg.misc?.versionary
+                if (stream_info.containsKey('versionary')) {
+                    // stream override always wins
+                    use_versionary = stream_info.versionary
+                }
```

Comment:
i'm guessing if we used this in one other place we'd put a helper function in `utils.groovy`?

Reply:
Yeah, agree. I don't anticipate that need for now.